src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.cs

```diff
@@ -18,6 +17,12 @@ internal sealed class DefaultAnalyzerAssemblyLoader : AnalyzerAssemblyLoader
     {
 #if NETCOREAPP
 
+        // Called from a netstandard2.0 project, so need to ensure a parameterless constructor is available.
+        internal DefaultAnalyzerAssemblyLoader()
```

Comment:
FYI @jaredpar

Let me know if there is a different solution here, that you'd prefer. The Razor EA project is netstandard2.0, so the compiler binds to the empty constructor, but at runtime in a netcoreapp project there was no empty constructor.

Reply:
This approach looks good to me

Reply:
This looks fine to me. Thanks for fixing this up.